libcxx/include/__exception/exception_ptr.h

```diff
@@ -94,38 +95,50 @@ class _LIBCPP_EXPORTED_FROM_ABI exception_ptr {
 template <class _Ep>
 _LIBCPP_HIDE_FROM_ABI exception_ptr make_exception_ptr(_Ep __e) _NOEXCEPT {
 #  if _LIBCPP_HAS_EXCEPTIONS
-#    if _LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION && __cplusplus >= 201103L
-  using _Ep2 = __decay_t<_Ep>;
-
-  void* __ex = __cxxabiv1::__cxa_allocate_exception(sizeof(_Ep));
+#    if _LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION && __cplusplus >= 201703L
```

Comment:
This should really be `_LIBCPP_STD_VER >= 17`. Also, why did this change from C++11 to C++17?

Reply:
This is about language-level features rather than libc++ features: c++11 or higher was required for the lambda to compile; c++17 or higher is now required for `if constexpr`

Reply:
Both are available as extensions in older versions. No need to put this behind a guard.